website/src/client/components/Preferences/PreferencesProvider.tsx

```diff
@@ -61,6 +62,16 @@ class PreferencesProvider extends React.Component<Props, State> {
 
     let overrides: Partial<PreferencesType> = {};
 
+    try {
+      // Restore editor preferences from local storage
+      if (localStorage) {
+        overrides.editorMode = localStorage.getItem('editorMode') as EditorModeType;
+        overrides.theme = localStorage.getItem('theme') as ThemeName;
```

Comment:
@Simek FYI, I'll add some defaults to this check. Right now it uses the wrong theme when users did not store any data yet. (wrong syntax highlighting)

Reply:
Oh, that's a bit surprising to me, but thanks for catching this. 👍 

I have tested similar scenario when I was checking that query params overwrites the local storage values IIRC, but probably I have missed this issue somehow. 😅